Makefile

```diff
@@ -73,7 +73,7 @@ requirements.txt: poetry.lock pyproject.toml
 format: setup $(files)
 	${PYTHON} tox -e fmt
 
-unittest: setup $(files)
+unittests: setup $(files)
```

Comment:
I personally preferred it as `unittest` but might've added a hyphen as `unit-test`